editor-components/feedback/feedbackmodal1/feedbackmodal1.tsx

```diff
@@ -0,0 +1,249 @@
+          alt=""  className={this.decorateCSS("icon")}/>
+          <h1 className={this.decorateCSS("feedbackModalHeaderh1")}>Feedback</h1>
+          <button className={this.decorateCSS("feedbackModalCloseButton")}>&times;</button>
+        </div>
+        <div className={this.decorateCSS("headerContent")}>
+          <h2>{this.getPropValue("headerTitle")}</h2>
+        </div>
+        <div className={this.decorateCSS("descriptionContent")}>
+          <p>{this.getPropValue("description")}</p>
+        </div>
+        <div className={this.decorateCSS("feedbackModalEmojis")}>
+          {this.castToObject<any[]>("emojiProps")?.map((item: any, index: number) => {
+            const label = item?.value?.[0]?.value;
+            const icon = item?.value?.[1]?.value;
+            const color = item?.value?.[2]?.value;
+
+            return (
+              <span
+                key={index}
+                id={`emoji-${label}`}
+                className={this.decorateCSS("feedbackModalEmoji")}
+                onClick={() => this.handleEmojiClick(label, color)}
+              >
+                <span className={this.decorateCSS("feedbackModalEmojiIcon")}>{icon}</span>     
```

Comment:
do not render relevant elements if there is no icon entered. do this operation for all props